test/e2e/main_test.go

```diff
@@ -83,6 +87,12 @@ func TestMain(m *testing.M) {
 		log.Fatal(err)
 	}
 
+	// The DEPLOY_KBS is exported then provisioner will install kbs before installing CAA
+	shouldDeployKbs := false
```

Comment:
Should the default be false? Or what is the harm in deploying KBS on all k8s by default?

Reply:
If cluster is being provisioned, then kbs also can be provisioned by default for e2e test.